packages/lesswrong/server/emailComponents/EmailUsernameByID.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { useSingle } from '../../lib/crud/withSingle';
+
+const EmailUsernameByID = ({userID}: {
```

Comment:
This is for [email notifications](https://github.com/ForumMagnum/ForumMagnum/pull/5335/files#r929396490)